tasks/engine_setup.yml

```diff
@@ -0,0 +1,60 @@
+---
+- block:
+  - name: Use the default answerfile
+    template:
+      src: answerfile_{{ ovirt_engine_setup_version }}_basic.txt.j2
+      dest: /tmp/answerfile.txt
```

Comment:
I am unsure about this hardcoded path, I think that maybe we should use some timestamp suffix.
like:

```yaml
- name: Set answer file path
  set_fact:
    answer_file_path: "/tmp/answerfile-{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}.txt"

 - name: Copy custom answer file
    template:
      src: "{{ ovirt_engine_setup_answer_file_path }}"
      dest: "{{ answer_file_path }}"
      mode: 0600
      owner: root
      group: root
    when: ovirt_engine_setup_answer_file_path is defined
```

Reply:
@machacekondra I can change it, but in the end of the playbook I remove it anyway.

Reply:
I know, but this file has too generic name, so it could be created by another app and  we rewrite it and remove it...

Reply:
Right, fixed!